src/index.js

```diff
@@ -113,6 +113,15 @@ const getDefaultLang = function GetDefaultLang () {
   return document.documentElement.getAttribute('lang') || 'en'
 }
 
+const getEditor = function GetEditor () {
+  const appNode = document.querySelector(APP_SELECTOR)
+  if (!appNode) {
+    console.warn(`Cozy-bar can't discover the app's Editor, and will probably fail to initialize the connection with the stack.`)
+    return ''
+  }
+  return appNode.dataset.cozyEditor || ''
```

Comment:
I personally have a preference for `null` instead of `undefined`. Something that should be in the guidelines ? 🗡